src/bin/cql-stress-cassandra-stress/java_generate/distribution/enumerated.rs

```diff
@@ -17,6 +17,13 @@ impl<T: Copy> EnumeratedDistribution<T> {
     pub fn sample(&self) -> T {
         self.items[self.dist.sample(&mut rand::thread_rng())].0
     }
+
+    pub fn has_item(&self, t: &T) -> bool {
+        self.items
+            .iter()
+            .map(|(item, _weight)| item)
+            .any(|item| item == t)
```

Comment:
Is the `map` really needed? We can do with one `any`.

Reply:
It is not, good catch